components/Buttons/ButtonContainer.js

```diff
@@ -0,0 +1,22 @@
+import SaveButton from "./SaveButton";
+import CancelButton from "./CancelButton";
+import { Container } from "./styles";
+
+export default function ButtonContainer({
+  boxesToAdd,
+  bucketsToAdd,
+  setCustomerData,
```

Comment:
instead of passing the setter function directly, you could define a handle function in the parent component instead and pass it down as a prop. You can then call the handle function to change the state :)

Reply:
Hey thanks, 

I know what you mean and normally would totally agree, 
but my next step is to install a database to the project, so the work i would do for this would be quite unnecessary